### ISSUE TITLE
Feature Request: Detailed Install Logs

### ISSUE THREAD

**Issue by ZafotheNinja** [TheOfficialFloW/VitaShell#528]
For the increasingly paranoid about their vita's stability, especially in a world where practically every homebrew requires unsafe access. I propose a way to find out exactly what a vpk unpacks and where, through installation logs. 

Is this homebrew only accessing ux0, and where it should? Or is it scattering bloat all over the system and modifying/overwriting things it shouldn't? Imho there should be some kind of monitoring and/or prevention in place so users don't have to worry about X homebrew bootlooping their system and undoing hours of work.

Tldr: Detailed install log and partition restriction options.
